### PR TITLE
Add option to rename adobe process files

### PR DESCRIPTION
**Pull request by ItsProfessional**
- Add option to rename adobe process files
  - Some adobe process files are mostly unnecessary yet they are always running in the background
  - Even if you disable these processes, the functionality of adobe programs will remain to 95%.
  - Some features of adobe apps might break but they are mostly unnoticeable and unused.
  - Perhaps, a warning should be shown to the user that some functionality of adobe programs might be lost, but that is for someone else to do, as I will leave it here
- Remove empty newline at the end of ServiceBlock.bat
- Remove usage of unnecessary %%B variable in for loops in HideCCFolder.bat

### CODE REVIEW

File: scripts/RenameFiles.bat [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+echo                  ^|                                                               ^| 
+echo                  ^|                                                               ^|
+echo                  ^|                            CCSTOPPER                          ^|
+echo                  ^|                         Made by eaaasun                       ^|
+echo                  ^|                        RenameFiles Module                     ^|
+echo                  ^|      ___________________________________________________      ^|
+echo                  ^|                                                               ^|
+echo                  ^|                  THIS WILL RENAME ADOBE FILES!                ^|
+echo                  ^|                                                               ^|
+echo                  ^|      It is HIGHLY recommended to create a system restore      ^|
```
Comment (eaaasun) [inline scripts/RenameFiles.bat:106]: Creating a system restore point will not backup the targeted files here

File: scripts/RenameFiles.bat [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+echo                  ^|                                                               ^| 
+echo                  ^|                                                               ^|
+echo                  ^|                            CCSTOPPER                          ^|
+echo                  ^|                         Made by eaaasun                       ^|
+echo                  ^|                        RenameFiles Module                     ^|
+echo                  ^|      ___________________________________________________      ^|
+echo                  ^|                                                               ^|
+echo                  ^|              Renaming adobe process files complete!           ^|
+echo                  ^|                                                               ^|
+echo                  ^|      The system needs to restart for changes to apply.        ^|
```
Comment (eaaasun) [inline scripts/RenameFiles.bat:167]: There's no need for the system to restart as long as all adobe processes were stopped before changing file names

Comment (ItsProfessional) [reply scripts/RenameFiles.bat:167]: I used the acrobat fix script as a base, and forgot to cut this part of the code out

File: scripts/RenameFiles.bat (hunk 1 above)
Comment (ItsProfessional) [reply scripts/RenameFiles.bat]: Added option for backing up adobe files instead of creating restore point

**Review by eaaasun — APPROVED** [review]

**Pull request merged**